packages/react-native/React/Views/UIView+React.h

```diff
@@ -104,6 +105,9 @@
 @property (nonatomic, readonly) UIEdgeInsets reactCompoundInsets;
 @property (nonatomic, readonly) CGRect reactContentFrame;
 
+@property (nonatomic, assign) CATransform3D reactTransform;
```

Comment:
Add a comment explaining this is the transform before any origin changes?

Also worth marking this as old-architecture only.

Reply:
Do you just want a comment or wrap `#ifndef RCT_NEW_ARCH_ENABLED`s around all this stuff. I suppose there is benefit to not adding the code, so the dynamic selector lookups are marginally cheaper

Reply:
No strong opinion on wrapping this with #ifdef, but a comment would be useful to make sure we remove this post the rollout of the new architecture.